site/docs/v1/tech/apis/webauthn.adoc

```diff
@@ -13,14 +13,23 @@ include::docs/v1/tech/shared/_advanced-edition-blurb.adoc[]
 This API has been available since 1.41.0
 ====
 
-This page contains all of the APIs for managing WebAuthn passkeys, sometimes referred to as credentials.
+This page contains all of the APIs for managing WebAuthn passkeys, sometimes referred to as credentials, and starting and completing WebAuthn ceremonies.
 
 The following APIs are provided to manage WebAuthn passkeys.
 
 * <<Retrieve a Passkey>>
 * <<Delete a Passkey>>
 * <<Import Passkeys>>
 
+=== WebAuthn Registration Ceremony
+* <<Start a WebAuthn Passkey Registration>>
+* <<Complete a WebAuthn Passkey Registration>>
+
+=== WebAuthn Authentication Ceremony
+* <<Start a WebAuthn Passkey Assertion>>
```

Comment:
Or maybe not.

Reply:
I wasn't sure how to phrase it. Both authentication and assertion use the same start API. I was worried about saying "Start a WebAuthn Passkey Authentication" and someone thinking "but I just want to do an assertion with the passkey grrrr."

I guess the same could be said for the way I have it now. I think using "Authentication" is better because that's what the ceremony is called.

Thoughts?

Reply:
How about `Start a WebAuthn Passkey Assertion or Authentication`?